charts/forms-flow-analytics/templates/deployment.yaml

```diff
@@ -17,281 +18,179 @@ spec:
-  strategy:
-  template:
-    metadata:
-      labels:
-        app: "{{ .Chart.Name }}-scheduler"
-    spec:
+      securityContext:
+        fsGroup: 2000 # Setting the file system group
       containers:
-      - args:
-        - scheduler
-        envFrom:
-        - configMapRef:
-            name: "{{ .Chart.Name }}" # default redash environment variables
-        - secretRef:
-            name: "{{ .Chart.Name }}" # default redash environment variables
-        env:
-        {{- if .Values.externalDatabase.existingDatabaseUrlKey }}
-        - name: REDASH_DATABASE_URL
-          valueFrom:
-            secretKeyRef:
-              key: {{ .Values.externalDatabase.existingDatabaseUrlKey }}
-              name: "{{ .Values.externalDatabase.existingSecretName }}"
-        {{- end }}
-        {{- if not .Values.externalDatabase.existingDatabaseUrlKey }}
-        - name: REDASH_DATABASE_URL
-          valueFrom:
-            configMapKeyRef:
-              key: REDASH_DATABASE_URL
-              name: "{{ .Chart.Name }}"
-        {{- end }}
-        image: "{{ .Values.redash.image.registry}}/{{ .Values.redash.image.repository }}:{{ .Values.redash.image.tag }}"
-        name: "{{ .Chart.Name }}-scheduler"
-        imagePullPolicy: Always
-        resources:
-          limits:
-            cpu: "{{ .Values.resources.limits.cpu }}"
-            memory: "{{ .Values.resources.limits.memory }}"
-          requests:
-            cpu: "{{ .Values.resources.requests.cpu }}"
-            memory: "{{ .Values.resources.requests.memory }}"
-      restartPolicy: Always
-      imagePullSecrets:
-      - name: "{{ .Values.formsflow.auth}}"
+        - name: '{{ .Chart.Name }}-scheduler'
+          image: '{{ .Values.redash.image.registry }}/{{ .Values.redash.image.repository }}:{{ .Values.redash.image.tag }}'
+          imagePullPolicy: Always
+          securityContext:
+            runAsNonRoot: true
+            runAsUser: 1000
+            readOnlyRootFilesystem: true
+            capabilities:
+              drop:
+                - ALL
+          args:
+            - scheduler
+          envFrom:
+            - configMapRef:
+                name: '{{ .Values.formsflow.configmap }}' # Default Redash environment variables
+            - secretRef:
+                name: '{{ .Chart.Name }}' # default redash environment variables
+          resources:
+            limits:
+              cpu: '{{ .Values.resources.limits.cpu }}'
+              memory: '{{ .Values.resources.limits.memory }}'
+            requests:
+              cpu: '{{ .Values.resources.requests.cpu }}'
+              memory: '{{ .Values.resources.requests.memory }}'
+
 ---
+{{- range $workerName, $config := .Values.workers }}
+{{- $workerConfig := mergeOverwrite (deepCopy $.Values.worker) $config }}
+{{- $context := mergeOverwrite (deepCopy $) (dict "workerName" $workerName) }}
+
 apiVersion: apps/v1
 kind: Deployment
 metadata:
+  name: forms-flow-analytics-worker
```

Comment:
in other templates it's using {chart name} - server, here it's hard coded. Please follow same pattern